pkg/unikontainers/utils.go

```diff
@@ -90,17 +100,22 @@ func moveFile(sourceFile string, targetDir string) error {
 // loadSpec returns the Spec found in the given bundle directory
 func loadSpec(bundleDir string) (*specs.Spec, error) {
 	var spec specs.Spec
-	absDir, err := filepath.Abs(bundleDir)
+
+	absBundleDir, err := filepath.Abs(bundleDir)
 	if err != nil {
-		return &spec, fmt.Errorf("failed to find absolute bundle path: %w", err)
+		return nil, fmt.Errorf("failed to find absolute path of bundle: %w", err)
 	}
-	data, err := os.ReadFile(filepath.Join(absDir, "config.json"))
+
+	configFile := filepath.Join(absBundleDir, configFilename)
+	specData, err := os.ReadFile(configFile)
 	if err != nil {
-		return &spec, fmt.Errorf("failed to read config.json: %w", err)
+		return nil, fmt.Errorf("failed to read specification file: %w", err)
 	}
-	if err := json.Unmarshal(data, &spec); err != nil {
-		return &spec, fmt.Errorf("failed to parse config.json: %w", err)
+
+	if err := json.Unmarshal(specData, &spec); err != nil {
+		return nil, fmt.Errorf("failed to parse specification json: %w", err)
```

Comment:
Since the `%w` verb allows for error wrapping, would it be a good practice to use across the whole codebase instead of using `%v`?

Reply:
Sure, we can do that, but are you referring to this particular PR or in general?

Reply:
In general, this is out of scope for this PR. But since you used it in this PR, I thought it would be a good opportunity to discuss this for the entirety of `urunc`.

Reply:
Yes, actually we already have in our TODOs to improve the logs in `urunc`. Therefore, it could be another good point to fix along with it.